hyperware-ai/hyperdrive
# Designed default app icons & appstore visuals

## Problem

- [x] change the "color dot" icons to deterministically show one of the designer's preset stock icons, on homepage and in appstore
- [x] appstore download/install/launch buttons are now all different colors to better show change of state in ui
- [x] appstore shows all apps on main page, paginated, and featured apps as chosen by Kinode
- [x] remove "ratings" from app page until feature is done
- [x] use the 🚫 mouse cursor for unclickable app icons

## Testing

- verify that system apps on homepage "all apps" have a designed icon and a 🚫 cursor if no web ui exists for it
- verify that you can search for apps on the appstore
- verify all listed apps show up in a scrollable view on the main appstore page
- verify ratings aren't shown on app pages anymore
- download an app with a ui. verify that the buttons for download, install, and launch are black/orange/white respectively

## Notes

There are some superfluous `@ts-ignore` comments in an appstore ui file - I think my IDE is bugged because the exact same file isn't showing those errors in the homepage ui. It compiles and runs as expected all the same.
